### PR TITLE
CMP-2998: added CO 1.6.1 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/CMP-2998

Link to docs preview:
[85469--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html](https://85469--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html)

QE review:
- [ ] QE has approved this change.

Additional information:
This update does not change the meaning of the docs; no QE review required.

### CODE REVIEW

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -15,6 +15,13 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-6-1_{context}"]
+== OpenShift Compliance Operator 1.6.1
+
+The following advisory is available for the OpenShift Compliance Operator 1.6.1:
+
+* link:https://access.redhat.com/errata/RHBA-2024:10367[RHBA-2024:10367 - OpenShift Compliance Operator update]
+
```
Comment (rhmdnd) [inline security/compliance_operator/compliance-operator-release-notes.adoc:24]: Do we need to add anything about upgrading dependencies in the underlying base images?

Comment (sheriff-rh) [reply security/compliance_operator/compliance-operator-release-notes.adoc:24]: I'll add the statement we've used previously that these updates address upgraded dependencies in underlying images. 👍

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Comment by sheriff-rh** [comment]
Not sure what happened here - I rebased and it closed my PR, saying it was merged. I verified the content was not merged and then created a clone: https://github.com/openshift/openshift-docs/pull/85957

**Pull request closed without merging**